### ISSUE TITLE
Accessibility: 

### ISSUE THREAD

**Issue by PreciousOritsedere** [jupyterhub/outreachy#58]
### What page is this for?

https://mybinder.org/

### WAVE accessibility report

![Screenshot (79)](https://user-images.githubusercontent.com/104203890/195921266-c3ae02dd-3c79-4330-be22-f6feebf94e1d.png)

### The HTML element to be changed

```html
1. <div id="banner-container">
<div style="display:flex;align-items:center;">
<div style="flex:1;text-align:center;">
Thanks to
Very low contrast
<a href="https://cloud.google.com/">
Google Cloud
</a>

,
Very low contrast
<a href="https://www.ovh.com/">
OVH
</a>

,
Very low contrast
<a href="https://notebooks.gesis.org">
GESIS Notebooks
</a>

and the
Very low contrast
<a href="https://turing.ac.uk">
Turing Institute
</a>

for supporting us! 🎉
</div>
Device dependent event handlerVery low contrast
<a class="btn" style="width: fit-content; height: fit-content; padding: 10px; background-color: rgb(230, 101, 129); color: white; font-weight: bold; margin: -8px 0px;" onmouseover="this.style.backgroundColor='#d15b75'" onmouseout="this.style.backgroundColor='#e66581'" href="https://numfocus.salsalabs.org/donate-to-binder" target="_blank">
🤍 Donate to mybinder.org!
</a>
</div>
</div>

2. <div id="logo-container">
Missing alternative text
<img id="logo" src="/static/logo.svg?v=fe52c40adc69454ba7536393f76ebd715e5fb75f5feafe16a27c47483eabf3311c14ed9fda905c49915d6dbf369ae68fb855a40dd05489a7b9542a9ee532e92b" width="390px">
</div>


3. <a class="btn" style="width: fit-content; height: fit-content; padding: 10px; background-color: rgb(230, 101, 129); color: white; font-weight: bold; margin: -8px 0px;" onmouseover="this.style.backgroundColor='#d15b75'" onmouseout="this.style.backgroundColor='#e66581'" href="https://numfocus.salsalabs.org/donate-to-binder" target="_blank"> 🤍 Donate to mybinder.org! </a>

4. <div class="container">
<div class="row text-center">
Heading level 3
<h3>
questions?
<br>

join the
<a href="https://discourse.jupyter.org/c/binder">
discussion
</a>

, read the
<a href="https://mybinder.readthedocs.io/en/latest/">
docs
</a>

, see the
<a href="https://github.com/jupyterhub/binderhub">
code
</a>
</h3>
</div>
</div>
```

### Your proposed HTML (or CSS), after change

```html
<nav>
     <div id="banner-container">
<div style="display:flex;align-items:center;">
<div style="flex:1;text-align:center;">
Thanks to
Very low contrast
<a href="https://cloud.google.com/">
Google Cloud
</a>

,
Very low contrast
<a href="https://www.ovh.com/">
OVH
</a>

,
Very low contrast
<a href="https://notebooks.gesis.org">
GESIS Notebooks
</a>

and the
Very low contrast
<a href="https://turing.ac.uk">
Turing Institute
</a>

for supporting us! 🎉
</div>
Device dependent event handlerVery low contrast
<a class="btn" style="width: fit-content; height: fit-content; padding: 10px; background-color: rgb(230, 101, 129); color: white; font-weight: bold; margin: -8px 0px;" onmouseover="this.style.backgroundColor='#d15b75'" onmouseout="this.style.backgroundColor='#e66581'" href="https://numfocus.salsalabs.org/donate-to-binder" target="_blank">
🤍 Donate to mybinder.org!
</a>
</div>
</div>
<nav>


2. <div id="logo-container">
Missing alternative text
<img id="logo" src="/static/logo.svg?v=fe52c40adc69454ba7536393f76ebd715e5fb75f5feafe16a27c47483eabf3311c14ed9fda905c49915d6dbf369ae68fb855a40dd05489a7b9542a9ee532e92b" width="390px" alt="binder logo">
</div>


3. <a class="btn" style="width: fit-content; height: fit-content; padding: 10px; background-color: rgb(230, 101, 129); color: white; font-weight: bold; margin: -8px 0px;" onmouseover="this.style.backgroundColor='#d15b75'" onfocus="this.style.backgroundColor='#d15b75'" onmouseout="this.style.backgroundColor='#e66581'" onfocus="this.style.backgroundColor='#e66581'" href="https://numfocus.salsalabs.org/donate-to-binder" target="_blank"> 🤍 Donate to mybinder.org! </a>

4. <footer>
   <div class="container">
<div class="row text-center">
Heading level 3
<h3>
questions?
<br>

join the
<a href="https://discourse.jupyter.org/c/binder">
discussion
</a>

, read the
<a href="https://mybinder.readthedocs.io/en/latest/">
docs
</a>

, see the
<a href="https://github.com/jupyterhub/binderhub">
code
</a>
</h3>
</div>
</div>
<footer>
```

**Comment by minrk** [comment]
THanks! Can you provide a little more detail about the issues being fixed? What issues are fixed by each change?

**Comment by sgibson91** [comment]
Thank you for your contribution! Since the contribution period is now over, we will close this issue.
